main-actions/src/main/scala/sbt/Package.scala

```diff
@@ -66,8 +68,10 @@ object Package {
     setVersion(main)
 
     val cachedMakeJar = inputChanged(cacheStoreFactory make "inputs") {
-      (inChanged,
-       inputs: Map[File, String] :+: FilesInfo[ModifiedFileInfo] :+: Manifest :+: HNil) =>
+      (
+          inChanged,
+          inputs: Map[File, String] :+: FilesInfo[ModifiedFileInfo] :+: Manifest :+: HNil
+      ) =>
```

Comment:
This almost makes the code look alien (non-Scala).

Reply:
this code looked bad before and it's not great after scalafmt either.

but at least now you know where the parens start and end because they're symmetrical at the same column.

I would fix this manually with a local type alias.